src/box/field_default_func.c

```diff
@@ -19,7 +19,8 @@ field_default_func_verify(struct func *func)
 {
 	const char *func_name = func->def->name;
 
-	if (func->def->language == FUNC_LANGUAGE_LUA) {
+	if (func->def->language == FUNC_LANGUAGE_LUA ||
+	    func->def->language == FUNC_LANGUAGE_SQL_EXPR) {
```

Comment:
There is some inconsistency with [tuple_constraint_func_verify](https://github.com/tarantool/tarantool/blob/f399cc7b7e2df43410ba71a60e4079723d39fa82/src/box/tuple_constraint_func.c#L45) — `func->def->body` is checked for NULL only for Lua there, not for the SQL. Anyway, you forgot to remove `TODO(gh-8793): Support SQL` below.

Reply:
Thanks, fixed!